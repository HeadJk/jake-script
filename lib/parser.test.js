"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const parser_1 = require("./parser");
const tokenizer_1 = require("./tokenizer");
test('it should pass', () => {
    expect((0, parser_1.parser)((0, tokenizer_1.tokenizer)('log(1+3)'))).toMatchSnapshot();
});
