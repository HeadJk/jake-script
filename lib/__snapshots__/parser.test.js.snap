// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should pass 1`] = `
Object {
  "body": Array [
    Object {
      "argument": Object {
        "kind": "BinaryExpressionNode",
        "left": Object {
          "kind": "NumericLiteralNode",
          "value": "1",
        },
        "operator": Object {
          "kind": "PlusToken",
        },
        "right": Object {
          "kind": "NumericLiteralNode",
          "value": "3",
        },
      },
      "identifier": Object {
        "kind": "IdentifierToken",
        "value": "log",
      },
      "kind": "CallExpressionNode",
    },
  ],
}
`;
